integration-tests/http/__tests__/shipping-option/admin/shipping-option.spec.ts

```diff
@@ -363,6 +379,280 @@ medusaIntegrationTestRunner({
+              ],
+            },
+            adminHeaders
+          )
+
+          expect(response.status).toEqual(200)
+
+          const updatedShippingOption = (
+            await api.get(
+              `/admin/shipping-options/${shippingOption.id}`,
+              adminHeaders
+            )
+          ).data.shipping_option
+          expect(updatedShippingOption).toEqual(
+            expect.objectContaining({
+              id: shippingOption.id,
+              rules: expect.arrayContaining([
+                expect.objectContaining({
+                  id: expect.any(String),
+                  operator: "eq",
+                  attribute: "old_attr",
+                  value: "old value",
+                  shipping_option_id: shippingOption.id,
+                }),
+                expect.objectContaining({
+                  id: expect.any(String),
+                  operator: "eq",
+                  attribute: "new_attr",
+                  value: "new value",
+                  shipping_option_id: shippingOption.id,
+                }),
+              ]),
+            })
+          )
+        })
+      })
+
+      describe("POST /admin/shipping-options/:id/rules/batch", () => {
+        it.skip("should throw error when required params are missing", async () => {
+          const shippingOptionPayload = {
+            name: "Test shipping option",
+            service_zone_id: fulfillmentSet.service_zones[0].id,
+            shipping_profile_id: shippingProfile.id,
+            provider_id: "manual_test-provider",
+            price_type: "flat",
+            type: {
+              label: "Test type",
+              description: "Test description",
+              code: "test-code",
+            },
+            prices: [
+              {
+                currency_code: "usd",
+                amount: 1000,
+              },
+              {
+                region_id: region.id,
+                amount: 1000,
+              },
+            ],
+            rules: [shippingOptionRule],
+          }
+
+          const shippingOption = (
+            await api.post(
+              `/admin/shipping-options`,
+              shippingOptionPayload,
+              adminHeaders
+            )
+          ).data.shipping_option
+
+          const { response } = await api
+            .post(
+              `/admin/shipping-options/${shippingOption.id}/rules/batch`,
+              {},
+              adminHeaders
+            )
+            .catch((e) => e)
+
+          expect(response.status).toEqual(400)
+          expect(response.data).toEqual({
+            type: "invalid_data",
+            message:
+              "each value in rule_ids must be a string, rule_ids should not be empty",
+          })
+        })
+
+        // FYI: This test was the only run being run for shipping option rules.
+        // TODO: The other tests should probably be updated?
+        it("should throw error when shipping option does not exist", async () => {
```

Comment:
Among the shipping option rules, this was the only test currently being run.

I have kept it this way and will get back later to see why other tests were skipped previously.